README.md

```diff
@@ -14,10 +14,10 @@
 <a href="https://discord.gg/cline" target="_blank"><strong>Join the Discord</strong></a>
 </td>
 <td align="center">
-<a href="https://github.com/cline/cline/wiki" target="_blank"><strong>Docs</strong></a>
+<a href="https://github.com/cline/cline/discussions/categories/feature-requests?discussions_q=is%3Aopen+category%3A%22Feature+Requests%22+sort%3Atop" target="_blank"><strong>Feature Requests</strong></a>
 </td>
 <td align="center">
-<a href="https://github.com/cline/cline/discussions/categories/feature-requests?discussions_q=is%3Aopen+category%3A%22Feature+Requests%22+sort%3Atop" target="_blank"><strong>Feature Requests</strong></a>
+<a href="https://cline.bot/join-us" target="_blank"><strong>We're Hiring!</strong></a>
```

Comment:
I don't think we actually want these links in the README for Roo-Cline

@mrubens thoughts?

Reply:
Yeah maybe we should just remove some of this stuff so as not to confuse people?

Reply:
from a git perspective, the merge is to baseline them

i don't have any opinions about content

git needs to record a history of the merge happening, then content can be updated aftewrwards

Reply:
without merging, i end up with differences trying to run git operations between the two different repositories because git sees a difference of a merge not happening yet regardless of content

Reply:
strangely, this pull request merged

but

https://github.com/RooVetGit/Roo-Cline/compare/main...cline%3Acline%3Amain

is still showing differences

and https://github.com/RooVetGit/Roo-Cline/

still says that roo cline is 1 commit behind cline

i don't understand

Reply:
is it because this merge was completed via a squash commit instead of a merge commit?  otherwise, i don't understand why github and git think that roo cline is 1 commit behind cline.